R/infrastructure.R

```diff
@@ -33,6 +39,12 @@ use_testthat <- function(pkg = ".") {
     pkg = pkg
   )
 
+  if (use_catch) {
```

Comment:
I don't think we need `use_rcpp()` for Catch (it doesn't depend on Rcpp); I think `use_catch()` should be able to handle this without also adding Rcpp infrastructure. Does that make sense?